test/unit/sdk.test.js

```diff
@@ -715,6 +715,58 @@ describe('sdk', () => {
 
       expect(link.includes('&pizza=isGood')).toBe(false);
     });
+
+    test('Adds single_select=true when singleSelect included', () => {
```

Comment:
what about when you do not include single_select? do you want to add that to one of the other tests or create a new test?

Reply:
I have added a test for this